charts/pega/README.md

```diff
@@ -658,6 +658,21 @@ If you are planning to use Cassandra (usually as a part of Pega Customer Decisio
 
 To use an existing Cassandra deployment, set `cassandra.enabled` to `false` and configure the `dds` section to reference your deployment.
 
+The following parameters are available for configuration of the connection to the external Cassandra cluster.
+
+Parameter     | Description                                                                                                                                                                                                                                                                                                                   | Default value
+---           |-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| ---
+`externalNodes`    | A comma separated list of hosts in the Cassandra cluster. | Empty
+`port` | TCP Port to connect to cassandra. | 9042
+`username` | The username for authentication with the Cassandra cluster. | dnode_ext
+`password` | The password for authentication with the Cassandra cluster. | dnode_ext
+`clientEncryption` | Whether to enable client encryption on the Cassandra connection. | false
+`trustStore` | If required, provide the trustStore certificate file name<br/>When using a trustStore certificate, you must also include a Kubernetes secret name that contains the trustStore certificate in the global.certificatesSecrets parameter.<br/>Pega deployments only support trustStores using the Java Key Store (.jks) format. | Empty
+`trustStorePassword` | If required provide trustStorePassword value in plain text. | Empty
+`keyStore` | If required, provide the keystore certificate file name<br/>When using a keystore certificate, you must also include a Kubernetes secret name that contains the keystore certificate in the global.certificatesSecrets parameter.<br/>Pega deployments only support keystore using the Java Key Store (.jks) format. | Empty
+`keyStorePassword` | If required provide keyStorePassword value in plain text. | Empty
```

Comment:
If required, provide a keyStorePassword value in plain text.

Let's think about this a little...this is what we said in the Kafka and cassandra areas in the Values yaml...>> # To avoid exposing trustStorePassword, keyStorePassword, and jaasConfig parameters, leave the values empty and
  # configure them using an External Secrets Manager, making sure you configure the keys in the secret ...
  
the point is to try and update the wording that it's best practice to not use the plain text passwords and instead  encrypt passwords in a secrets and manage it in external secrets operator.

Reply:
I will add a note that the better practice is to use a secrets manager rather than plain text password